docs/pages/blog/mui-x-end-v8.md

```diff
@@ -0,0 +1,243 @@
+
+Reorder grouping levels on the fly and stay in place. Expanded groups stay open even as data updates, so you don't lose context during refreshes:
+
+<figure style="margin:16px 0;">
+  <video src="https://github.com/user-attachments/assets/98da078f-0747-4a02-b304-6a66c936c995" autoplay muted loop playsinline controls style="width:100%;max-width:960px;border-radius:12px;"></video>
+  <figcaption style="text-align:center;margin-top:8px;">Drag to reorder groups; expansion persists.</figcaption>
+</figure>
+
+[Row grouping guide](/x/react-data-grid/row-grouping/)
+
+### Charts integration [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+Turn selections into visuals in seconds. An integrated panel renders the right chart for the current view and stays in sync with Grid interactions—ideal for quick investigations and stakeholder snapshots:
+
+<figure style="margin:16px 0;">
+   <video src="https://github.com/user-attachments/assets/1dde7294-5b7d-4862-b250-6299aaa29c4c" autoplay muted loop playsinline controls style="width:100%;max-width:960px;border-radius:12px;"></video>
+  <figcaption style="text-align:center;margin-top:8px;">One click from table to chart.</figcaption>
+</figure>
+
+[Charts integration docs](/x/react-data-grid/charts-integration/)
+
+### Smoother reordering with clear affordances [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+Drag-and-drop row reordering now shows a precise drop indicator, so teams can reprioritize with confidence—perfect for backlogs and ranked lists:
+
+<figure style="margin:16px 0;">
+  <video src="https://github.com/user-attachments/assets/7b6d7de3-70e9-4754-93c8-d7963d4b464a" autoplay muted loop playsinline controls style="width:100%;max-width:960px;border-radius:12px;"></video>
+  <figcaption style="text-align:center;margin-top:8px;">A clear target as you drag rows.</figcaption>
+</figure>
+
+[Row reordering docs](/x/react-data-grid/row-ordering/)
+
+### Pinned areas and scrolling polish
+
+Pinned rows and aggregates cooperate with the scrollbar and scroll shadows making wide, tall tables easier to parse at a glance:
+
+<figure style="margin:16px 0;">
+  <video src="https://github.com/user-attachments/assets/6aa59911-d247-48c5-b576-e3fb15092e7a" autoplay muted loop playsinline alt="Pinned rows and scroll shadows" style="width:100%;max-width:960px;border-radius:12px;">
+  <figcaption style="text-align:center;margin-top:8px;">Pinned context that reads at a glance.</figcaption></video>
+</figure>
+
+[Pinning guide](/x/react-data-grid/column-pinning/) • [Scrolling guide](/x/react-data-grid/scrolling/)
+
+### Export resilience [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+Excel export now relies on our own internally vetted and private `ExcelJS` fork to avoid transitive vulnerabilities and keep exports reliable at scale:
```

Comment:
```suggestion
Excel export now relies on our own internally vetted and private ExcelJS fork to avoid transitive vulnerabilities and keep exports reliable at scale:
```